networkit/cpp/components/test/ConnectedComponentsGTest.cpp

```diff
@@ -585,4 +586,18 @@ TEST_F(ConnectedComponentsGTest, testExtractLargestStronglyConnectedComponent) {
     }
 }
 
+TEST_F(ConnectedComponentsGTest, testRandomSpanningForest) {
+    Graph G(5);
```

Comment:
Better create a test graph in such a way, that a subset of edges with constant size is needed for creating a spanning tree. You can then also test for the number of edges in the spanning tree.